src/shared/components/VideoPreviewBase/VideoPreviewBase.tsx

```diff
@@ -197,17 +261,39 @@ const VideoPreviewBase: React.FC<VideoPreviewBaseProps> = ({
                   </Anchor>
                 ))}
               {showMeta && (
-                <MetaContainer main={main}>
+                <MetaContainer noMarginTop={showChannel === false} main={main}>
                   {isLoading ? (
                     <SpacedPlaceholder height={main ? 16 : 12} width={main ? '40%' : '80%'} />
                   ) : createdAt ? (
                     <MetaText variant="subtitle2" main={main} scalingFactor={scalingFactor}>
-                      {formatVideoViewsAndDate(views ?? null, createdAt, { fullViews: main })}
+                      {/* TODO: instead of createdAt add the lastUpdate from useDraft hook */}
```

Comment:
I think we can just leave it out as it is. We can assume that `createdAt` in case of a draft means `updatedAt`

```suggestion
```